src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationOptions.cs

```diff
@@ -82,22 +86,25 @@ public LinkedInAuthenticationOptions()
         /// <returns>The property value.</returns>
         private string GetMultiLocaleString(JObject user, string propertyName)
         {
-            const string localizedKey = "localized";
-            if (user[propertyName] == null || user[propertyName][localizedKey] == null)
+            var property = user[propertyName];
+            var propertyLocalized = property["localized"];
```

Comment:
You're getting this value before you do the `null` check, so it would throw a `NullReferenceException` if it were.

Reply:
🤦‍♂ me...

Reply:
Hopefully you would squash merge 😉